schemas/service.yml

```diff
@@ -105,3 +105,13 @@
 
         This id normally changes across restarts, but `service.id` does not.
       example: 8a4f500f
+
+    - name: address
+      level: extended
+      type: keyword
+      short: Address of this service.
+      description: > 
+        Address where this service is being run from.
+
+        This could be a URL or a filepath.
```

Comment:
Under what circumstances would this be a file path? Is it for a service listening on a unix socket or named pipe?

Reply:
Kylie is right here: [kvm](https://github.com/elastic/beats/blob/23e4403ae093fcc8f7905345cad2c7ad256976d8/metricbeat/module/kvm/status/_meta/data.json#L20)

Reply:
I think the description should be better now after updating it per the comment below. If you think there's still more to explain, lmk